src/components/MessageView.tsx

```diff
@@ -29,6 +29,7 @@ import {
 import NameView from './NameView'
 import { DispatchContext, UserMapContext } from '../App'
 import { deleteMessage, fetchProfile, moveToRoom } from '../networking'
+import { roomData } from '../../server/src/rooms'
```

Comment:
I'm not sure if there's a better way to handle this part, or if there's a worry about referencing a server file from this more client side. It does seem to behave properly.